microcloud/test/includes/microcloud.sh

```diff
@@ -466,15 +471,11 @@ reset_snaps() {
     fi
 
     echo "Enabling LXD and MicroCloud for ${name}"
-    lxc exec "${name}" -- sh -c "
-      snap enable lxd > /dev/null 2>&1 || true
-      snap enable microcloud > /dev/null 2>&1 || true
-      snap start lxd > /dev/null 2>&1 || true
-      snap start microcloud > /dev/null 2>&1 || true
-      snap refresh lxd --cohort=+
-
-      lxd waitready
-    "
+    lxc exec "${name}" -- snap enable lxd > /dev/null 2>&1 || true
+    lxc exec "${name}" -- snap enable microcloud > /dev/null 2>&1 || true
+    lxc exec "${name}" -- snap start lxd > /dev/null 2>&1 || true
+    lxc exec "${name}" -- snap start microcloud > /dev/null 2>&1 || true
```

Comment:
I just found that `snap start --enable` is a thing. Would that work here?

Reply:
I think they do different things.

`snap enable` re-enables a previously disabled snap, which treats it as though it's not installed.

`snap start --enable` just makes the snap start automatically next time the system is booted.